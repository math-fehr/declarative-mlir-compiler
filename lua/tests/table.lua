local a = { 0, 1, 2, 3, 4, 5 }
print(a[1])
print(a[6])
print(a[7])

local b = { 0, "a", { false } }
print(b[1])
print(b[2])
print(b[3][1])

