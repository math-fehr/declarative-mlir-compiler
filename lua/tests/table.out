0       
5       
nil     
0       
a       
false   
